### DISCUSSION TITLE
Using Cypress in an evironment with no access to the internet

### DISCUSSION THREAD

**Discussion by JohnPKent** [cypress-io/cypress#9445] [Questions and Help]
Hi
I'm using Cypress within a secure environment which has no internet connection.  What will not work for me apart from Dashboard and help?  Anything else I should know about?
Thanks

**Comment by bahmutov** [comment]
As long as you have downloaded Cypress or installed it, there should be nothing else necessary, you can use Cypress without any internet connection.

**Reply by JohnPKent** [reply]
Thanks

**Comment by Actvist** [comment]
OK....how?  :o)

**Reply by san-slysz** [reply]
What do you mean 'how'? There is nothing to do.
If the machine running the test has Cypress set up on it, and if baseUrl is reachable -and there's so many way to acheive that, talk to you CTO or devOps to just put the right architecutre for your product and pipeline- then it just works. Cypress is not making calls you did not ask it to do.

This said, do not confuse _Cypress_ for _Cypress Dashboard_ ;).

**Comment by san-slysz** [comment]
You also need your product to be reachable, it goes without saying.
